lib/fizzy/parser.hpp

```diff
@@ -140,4 +140,22 @@ struct parser<Code>
         return result;
     }
 };
+
+template <>
+struct parser<Data>
+{
+    parser_result<Data> operator()(const uint8_t* pos)
+    {
+        MemIdx memidx;
+        std::tie(memidx, pos) = leb128u_decode<uint32_t>(pos);
+
+        unsigned offset;
+        std::tie(offset, pos) = leb128u_decode<uint32_t>(pos);
+
+        std::vector<uint8_t> init;
+        std::tie(init, pos) = parser<std::vector<uint8_t>>{}(pos);
+
+        return {{memidx, offset, bytes{init.data(), init.size()}}, pos};
```

Comment:
I would use parentheses here not to look like it creates `bytes` with two elements
(actually I'm not sure why it doesn't)
```suggestion
        return {{memidx, offset, bytes(init.data(), init.size())}, pos};
```